vllm/model_executor/weight_utils.py

```diff
@@ -20,6 +21,9 @@
 from vllm.model_executor.layers.quantization import (get_quantization_config,
                                                      QuantizationConfig)
 
+if os.getenv('TENSORIZER_IGNORE_WARNINGS'):
+    warnings.filterwarnings("ignore")
```

Comment:
This is not an acceptable way to silence a single warning message! **This disables all warnings for all libraries for the entire program!** It's also not a warning from tensorizer, it's a warning from vLLM code (\<you are here\>), so it shouldn't be prefixed by `TENSORIZER_`. It'd be better as something like `VLLM_SILENCE_PERFORMANCE_WARNINGS`.
```suggestion
class PerformanceWarning(UserWarning):
    def __str__(self):
        return (
            f"{super().__str__()}" 
            " (set the VLLM_SILENCE_PERFORMANCE_WARNINGS"
            " environment variable to hide this)"
        )


if (
    os.getenv("VLLM_SILENCE_PERFORMANCE_WARNINGS", "").lower()
    not in ("", "0", "n", "no", "off", "disable")
):
    warnings.simplefilter("ignore", category=PerformanceWarning)
```